src/styles/blocks/util/variables.scss

```diff
@@ -0,0 +1,10 @@
+$roboto: "Roboto", sans-serif;
+
+$accent-color: #00acdc;
+$gray: #f3f3f3;
```

Comment:
Try not to name variables by content, but name them by meaning, as you did for other variables.